app/components/email-input.js

```diff
@@ -0,0 +1,66 @@
+import Component from '@ember/component';
+import { empty } from '@ember/object/computed';
+
+export default Component.extend({
+    type: '',
+    value: '',
+    isEditing: false,
+    user: null,
+    disableSave: empty('user.email'),
+    notValidEmail: false,
+    prevEmail: '',
+    emailIsNull: true,
+
+    actions: {
+        editEmail() {
+            let email = this.get('value');
+            let isEmailNull = function(email) {
+                if (email == null) {
```

Comment:
I think this might be special for models? I'm not sure exactly how it works on line 9 but when I print out the result of empty it isn't just a boolean type.